libs/utils/executor.py

```diff
@@ -512,6 +512,12 @@ def _wload_run(self, exp_idx, experiment):
         logging.debug(r'%14s - out_dir [%s]', 'Executor', experiment.out_dir)
         os.system('mkdir -p ' + experiment.out_dir)
 
+        # ANDROID: Stop all the Android system services that introduce noise
+        pause_android_services = tc.get("pause_android_services", True)
```

Comment:
Do you mean a target conf flag? I don't think this belongs in the TestEnv target configuration - it should be configurable per-experiment (also TestEnv doesn't have an "end" where the change can be undone). Will try with a target configuration flag.